README.md

```diff
@@ -48,56 +48,36 @@ Once your account is approved, contact us with your username and ask to be added
 
 Below is a quick reference guide for standard installation.  For more complete details and troubleshooting, see the [Full Installation Guide](doc/kb_sdk_dependencies.md).
 
-For the adventurous, there is an experimental Dockerized installation that only requires Docker.  It is known to work on at least Mac and Linux, but has not been extensively tested yet.  For details, see [Dockerized Installation](doc/kb_sdk_dockerized_install.md)
-
 #### Installation Only
 
 System Dependencies:
 
 - Mac OS X 10.8+ or Linux. kb-sdk does not run natively in Windows, but see [here](doc/FAQ.md#windows) for more details.
-- Java JRE 7 or 8 (9 is currently incompatible) http://www.oracle.com/technetwork/java/javase/downloads/index.html
 - (Mac only) Xcode https://developer.apple.com/xcode
 - git https://git-scm.com
 - Docker https://www.docker.com (for local testing)
 
-Get the SDK:
-
-    git clone https://github.com/kbase/kb_sdk
 
-Pull dependencies and configure the SDK:
+Pull the KBase SDK image.
 
-    cd kb_sdk
-    make bin
+    docker pull kbase/kb-sdk
 
-Download the local KBase SDK base Docker image:
+Create a wrapper script to call the SDK.  This script can be placed anywhere you wish, but it should be
+on your path.  In this example, we will install it in $HOME/bin/.  To use this approach on Windows, you may need to install the Windows for Linux (WSL).
 
-    make sdkbase 
+    docker run kbase/kb-sdk genscript > $HOME/bin/kb-sdk
+    chmod 755 $HOME/bin/kb-sdk
+    export PATH=$PATH:$HOME/bin/kb-sdk
 
-Add the kb-sdk tool to your PATH and enable command completion.  From the kb_sdk directory:
+Download the local KBase SDK base Docker image:
 
-    # for bash
-    export PATH=$(pwd)/bin:$PATH
-    source src/sh/sdk-completion.sh
+    kb-sdk sdkbase
```

Comment:
Should this be sdkbase2? Not sure what sdkbase means here

Reply:
It just triggers the pull.  We could use something like pullbase.

Reply:
yeah, I think a name that better describes what the command does (`pull-base-image` is exact but verbose) would be helpful.